osf/models/preprint_provider.py

```diff
@@ -128,8 +128,22 @@ def rules_to_subjects(rules):
             q.append(models.Q(_id=sub))
     return Subject.objects.filter(reduce(lambda x, y: x | y, q)) if len(q) > 1 else (Subject.objects.filter(q[0]) if len(q) else Subject.objects.all())
 
-
 @receiver(post_save, sender=PreprintProvider)
 def create_provider_auth_groups(sender, instance, created, **kwargs):
     if created:
         GroupHelper(instance).update_provider_auth_groups()
+
+
+class SharePreprintProviderWhitelisted(models.Model):
+    id = models.AutoField(primary_key=True)
+    provider_name = models.CharField(default=None, blank=True, unique=True, max_length=200)
```

Comment:
If it's not nullable, `default=None` will error. Probably don't want a default at all?

Reply:
Also not sure we want `blank=True`, this seems like the sort of thing that should always have a value.